KS.Fiks.IO.Arkiv.Client/ForenkletArkivering/ArkivmeldingFactory.cs

```diff
@@ -109,6 +114,10 @@ public static Arkivmelding GetArkivmelding(OppdaterSaksmappe input)
                     
                     var dok = new Dokumentobjekt
                     {
+                        Format = "X", //TODO Her mangler noe eller det er noe som ikke stemmer.
```

Comment:
Hva skal være standard? Eller skal det komme som inn-parameter i OppdaterSaksmappe?

Reply:
Noark 5 angir ikke noe her, men Arkade 5 som benyttes for test av avlevering forutsetter at det er filtypen, f.eks. "pdf".